### PR TITLE
relax activesupport version constraint to upgrade our app to rails 5+

### PR DESCRIPTION
**Pull request by harley**
cc @anvox @T-800 @dinhhuydh

### CODE REVIEW

**Comment by anvox** [comment]
You may need update appraisal config to run test against rails 5.2

**Comment by harley** [comment]
thanks @anvox. updated for tests to pass. does it look enough? do we need different appraisal groups for 5.1 and 5.2 separately?

File: lib/tp_common/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module TpCommon
-  VERSION = "0.2.2"
+  VERSION = "0.3.2"
```
[the remark this answers is not in the document]
Comment (harley) [reply lib/tp_common/version.rb]: oops haha thanks

**Comment by harley** [comment]
Thanks @anvox -- i'll merge now and I see it should auto deploy as you set up.

We'll have a backporting PR in our apps with tp_common upgrade -- will cc you for that.

**Comment by anvox** [comment]
@harley you need to tag master with version to release. I removed auto release to avoid merging mistake :D

**Pull request merged**